plugins/GSdx/GSDeviceDX.h

```diff
@@ -180,12 +170,14 @@ class GSDeviceDX : public GSDevice
 				uint32 point_sampler:1;
 				uint32 shuffle:1;
 				uint32 read_ba:1;
+
+				uint32 _free:32;
```

Comment:
Oh my bad you got 32 bits exactly. It would still be useful for future addition anyway :+1: